model/task.js

```diff
@@ -301,3 +302,12 @@ module.exports = function(app, callback) {
 		callback(errors, model);
 	});
 };
+
+function possibleCsvListToArray(value) {
+	if (Array.isArray(value)) {
+		return value;
+	} else if (typeof value === 'string') {
+		return value.split(',').forEach(item => item.trim());
```

Comment:
You need to use `map` to project the values from the `split` array. 

```suggestion
		return value.split(',').map(item => item.trim());
```